package/src/skia/web/api/JsiSkPath.ts

```diff
@@ -0,0 +1,342 @@
+import type { CanvasKit, Path } from "canvaskit-wasm";
+
+import type {
+  FillType,
+  PathCommand,
+  PathOp,
+  SkMatrix,
+  SkPath,
+  SkPoint,
+  SkRect,
+  SkRRect,
+  StrokeOpts,
+} from "../../types";
+import { PathVerb } from "../../types";
+import type { SkFont } from "../../types/Font/Font";
+
+import {
+  ckEnum,
+  HostObject,
+  NotImplementedOnRNWeb,
+  optEnum,
+  toValue,
+} from "./Host";
+import { JsiSkPoint } from "./JsiSkPoint";
+import { JsiSkRect } from "./JsiSkRect";
+
+export class JsiSkPath extends HostObject<Path, "Path"> implements SkPath {
+  constructor(CanvasKit: CanvasKit, ref: Path) {
+    super(CanvasKit, ref, "Path");
+  }
+
+  addArc(
+    oval: SkRect,
+    startAngleInDegrees: number,
+    sweepAngleInDegrees: number
+  ) {
+    this.ref.addArc(toValue(oval), startAngleInDegrees, sweepAngleInDegrees);
+    // TODO: fix in SkPath
```

Comment:
It doesn't seem like the WASM code returns the path, maybe it is just the types in CanvasKit?:

https://github.com/google/skia/blob/b2a8a7e3e89bfc39ae87459723ba8352ffa9933a/modules/canvaskit/canvaskit_bindings.cpp#L1545

Reply:
The TODO is about making it consistent (in either direction).